rpc/grpc/server/src/request_handler/factory.rs

```diff
@@ -122,6 +127,10 @@ impl Factory {
             });
         interface.replace_method(KaspadPayloadOps::NotifyFinalityConflict, method);
 
+        // Methods with special properties
+        let bps = bps as usize;
+        interface.replace_method_properties(KaspadPayloadOps::SubmitBlock, bps, 10.max(bps * 2), RoutingPolicy::DropIfFull);
```

Comment:
`tasks` is directly set using bps. Should there be some upper bound for `tasks` (consequently the handlers which cares about this number) or is it expected that it must grow linearly with bps?

Reply:
This will need further fine tuning when we explore higher BPS. For now, it's ok like that I guess.